src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksUtilityServiceImpl.java

```diff
@@ -164,6 +167,32 @@ public String trimFieldToMax(String field, String fieldName) {
 	public String convertPhoneNumber(String phoneNumber) {
 		return getPhoneNumberService().formatNumberIfPossible(phoneNumber);
 	}
+	
+	@Override
+	public boolean shouldVendorBeSentToPaymentWorks(VendorDetail vendorDetail) {
+		boolean sendToPaymentWorks = vendorDetail.isActiveIndicator()
+				&& !(StringUtils.equals(vendorDetail.getVendorHeader().getVendorOwnershipCategoryCode(), "US")
+						|| StringUtils.equals(vendorDetail.getVendorHeader().getVendorOwnershipCategoryCode(), "NU")
+						|| StringUtils.equals(vendorDetail.getVendorHeader().getVendorOwnershipCategoryCode(), "SE")
+						|| StringUtils.equals(vendorDetail.getVendorHeader().getVendorOwnershipCategoryCode(), "UE")
+						|| StringUtils.equals(vendorDetail.getVendorHeader().getVendorOwnershipCategoryCode(), "AE"));
+		
+		LOG.debug("shouldVendorBeSentToPaymentWorks1, sendToPaymentWorks: " + sendToPaymentWorks);
+		return sendToPaymentWorks;
+	}
+	
+	@Override
+	public boolean shouldVendorBeSentToPaymentWorks(PaymentWorksVendor paymentWorksVendor) {
+		Integer vendorNumber = paymentWorksVendor.getVendorHeaderGeneratedIdentifier();
+		Integer vendorParentChildNumber = paymentWorksVendor.getVendorDetailAssignedIdentifier();
+		VendorDetail vendorDetail = getVendorService().getVendorDetail(vendorNumber, vendorParentChildNumber);
+		if (ObjectUtils.isNotNull(vendorDetail)) {
+			return shouldVendorBeSentToPaymentWorks(vendorDetail);
+		} else {
```

Comment:
the else is not really needed here since the code after won't be called if the if condition is true and the method returns.